acceptance/lib/beaker/acceptance/install_utils.rb

```diff
@@ -0,0 +1,58 @@
+module Beaker
+  module Acceptance
+    module InstallUtils
+
+      PLATFORM_PATTERNS = {
+        :redhat        => /fedora|el|centos/,
+        :debian        => /debian|ubuntu/,
+        :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
+        :solaris_10    => /solaris-10/,
+        :solaris_11    => /solaris-11/,
+        :windows       => /windows/,
+        :sles          => /sles/,
+      }.freeze
+
+      # Installs packages on the hosts.
+      #
+      # @param hosts [Array<Host>] Array of hosts to install packages to.
+      # @param package_hash [Hash{Symbol=>Array<String,Array<String,String>>}]
+      #   Keys should be a symbol for a platform in PLATFORM_PATTERNS.  Values
+      #   should be an array of package names to install, or of two element
+      #   arrays where a[0] is the command we expect to find on the platform
+      #   and a[1] is the package name (when they are different).
+      # @param options [Hash{Symbol=>Boolean}]
+      # @option options [Boolean] :check_if_exists First check to see if
+      #   command is present before installing package.  (Default false)
+      # @return true
+      def install_packages_on(hosts, package_hash, options = {})
+        return true if hosts == nil
+        check_if_exists = options[:check_if_exists]
+        hosts = [hosts] unless hosts.kind_of?(Array)
+        hosts.each do |host|
```

Comment:
thanks Rick.  There is so much that should be done in here, that i hesitate to do anything at all.  This file was actually just moved (from: acceptance/pre_suite/puppet_git/install.rb, below) and is really only used as a fixture for the smoke tests.  We should probably be pulling the tests from the puppet and other project repos to stay up to date... 
in addition, this method really should be merged into beaker-proper.  IMO, this should be ticketed separately as part of the greater merge-up tech-debt effort.